homeassistant/components/luxtronik/binary_sensor.py

```diff
@@ -0,0 +1,124 @@
+"""
+Support for monitoring the state of a Luxtronik heatpump.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/integrations/luxtronik/
```

Comment:
Please remove these two lines since they are not describing what the file is about. See [file headers](https://developers.home-assistant.io/docs/en/development_guidelines.html#file-headers) for an example.